package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "prepublish": "npm run test",
-    "test": "mocha"
+    "test": "node ./src/extensions/core/index.spec.js"
```

Comment:
This is temporary. We will use mocha again in the future, when i have time to port over all tests to the new structure